tools/codegen/dest/lazy_ir.py

```diff
@@ -105,7 +105,7 @@ def gen(self, f: Union[NativeFunctionsGroup, NativeFunction]) -> List[str]:
         node_ctor_args = ", ".join([f"const {i.cpp_type()}& {i.name}" for i in all_types])
         scalar_initializers = ",\n        ".join([f"{t.name}({t.name})" for t in scalar_types])
         comma_if_scalar_initializers = ",\n" if len(scalar_initializers) else ""
-        scalar_decls = "\n  ".join([f"{t.cpp_type()} {t.name};" for t in scalar_types])
+        scalar_decls = "\n  ".join([f"std::string {t.name};" if t.cpp_type() == "c10::string_view" else f"{t.cpp_type()} {t.name};" for t in scalar_types])
```

Comment:
i do want the lint to pass so the line can't be too long

more importantly, i think we should probably fix this inside the data model (basically, t.cpp_type() should probably return std::string for us)

its fine to land this without updating the data model, but it might be easy to do that at the same time.

please TAL at my refactor PR https://github.com/pytorch/pytorch/pull/73939 - it might be easier to modify the data model after that PR lands, or reviewing it might help your understanding of the data model

Reply:
I will break the line to pass lint.

Having t.cpp_type() return c10::string_view seems more consistent with other types. E.g. there is another call to cpp_type() at Line 105, which result seems more natural because the parameter is declared as `const c10::string_view&` and the passed-in variable is also declared as c10::string_view at the callsite in core.

Reply:
I will merge the diff now and update it later.